sycl/doc/syclcompat/README.md

```diff
@@ -197,6 +197,44 @@ The second interface allows users to allocate device local memory at runtime.
 SYCLcompat provides this functionality through its kernel launch interface,
 `launch<function>`, defined in the following section.
 
+Finally, the following cuda backend specific functions are introduced in order
```

Comment:
```suggestion
The following cuda backend specific functions are introduced in order
```
Just to slightly reduce the maintenance burden when we add more stuff :sweat_smile: